test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -59,6 +67,10 @@
 	public static OptionalInt totalTestTokenAccounts = OptionalInt.empty();
 	public static OptionalInt memoLength = OptionalInt.of(DEFAULT_MEMO_LENGTH);
 
+	protected final ResponseCodeEnum[] standardPermissiblePrechecks = new ResponseCodeEnum[] {
+			OK, BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED
```

Comment:
Should `PLATFORM_NOT_ACTIVE` be one of them?

Reply:
As `PLATFORM_NOT_ACTIVE` is not a permissible precheck, and the test should be stopped for that status, it is not added.

Reply:
I see your point. I was thinking there might be two scenarios that `PLATFORM_NOT_ACTIVE` might be a permissible precheck: 1) The client comes up slightly earlier than server node. If the client can retry several times, the test can still be successful; 2) The server node may be temporarily unavailable due to network glitch, the retry will be still helpful.

Reply:
@ljianghedera Got it. In my opinion for 
1. we need to use JRS setting to make sure clients start after server as this is Engineering testing
2. Though this is a a valid case, this case should not happen mostly, unless we are simulating Restart/Reconnect and trying to submit transactions to the node that is being restarted.